### DISCUSSION TITLE
Port Forwarding Not working properly????

### DISCUSSION THREAD

**Discussion by SamlyBoy11255** [itzg/docker-minecraft-server#2455] [Q&A]
my minecraft server is only accesable using LAN and on a different wifi it says my server doesnt even exist. I obviously before this had went into my wifi settings and enabled port forwarding for my server IP. wondering if it could be my WAN mode, Security type, or something i have to change in my docker container.  I also checked https://canyouseeme.org and it said it saw my port.

**Comment by casudo** [comment]
Did you enter the correct public IP address? Is the correct port published in your container?
